integration-libs/opf/global-functions/root/model/opf-global-functions.model.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ViewContainerRef } from '@angular/core';
+import { KeyValuePair } from '@spartacus/opf/base/root';
+
+export enum GlobalFunctionsDomain {
+  CHECKOUT = 'checkout',
+  GLOBAL = 'global',
```

Comment:
[QUESTION]

I have problem to find the usages of the value `GLOBAL`. Is it deliberate? Or we can simply remove it?